### PR TITLE
[Owl] Fix Lessons FAB being behind nav bar.

### PR DESCRIPTION
**Pull request by nickbutcher**

### CODE REVIEW

File: Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt [hunk 1]
```diff
@@ -125,7 +125,11 @@ fun CourseDetails(
                     orientation = Orientation.Vertical
                 )
             ) {
-                val openFraction = -sheetState.offset.value / dragRange
+                val openFraction = if (sheetState.offset.value.isNaN()) {
```
Comment (chrisbanes) [inline Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt:128]: Does `isNan()` ever happen?

File: Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt [hunk 2]
```diff
@@ -316,8 +320,9 @@ private fun LessonsSheet(
 ) {
     // Use the fraction that the sheet is open to drive the transformation from FAB -> Sheet
     val fabSize = with(DensityAmbient.current) { FabSize.toPx() }
+    val fabHeight = fabSize + InsetsAmbient.current.systemBars.bottom
```
Comment (chrisbanes) [inline Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt:323]: Nit: Technically the insets aren't part of the FAB height.

File: Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt (hunk 1 above)
Comment (nickbutcher) [reply Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt:128]: Yes, `Swipeable` changed to [initialize this offset with `NaN`](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:compose/material/material/src/commonMain/kotlin/androidx/compose/material/Swipeable.kt;l=95) so this actually fixes a run time crash.

File: Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt (hunk 2 above)
Comment (nickbutcher) [reply Owl/app/src/main/java/com/example/owl/ui/course/CourseDetails.kt]: Done.

**Review by manuelvicnt — APPROVED** [review]

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**